src/libs/runtime-utils.js

```diff
@@ -53,16 +59,22 @@ export const normalizeRuntimeConfig = ({
     // One caveat with using DEFAULT_BOOT_DISK_SIZE here is this over-estimates old GCE runtimes without PD by 1 cent
     // because those runtimes do not have a separate boot disk. But those old GCE runtimes are more than 1 year old if they exist.
     // Hence, we're okay with this caveat.
-    bootDiskSize: bootDiskSize || defaultGceBootDiskSize
+    bootDiskSize: bootDiskSize || defaultGceBootDiskSize,
+    computeRegion: computeRegion || defaultComputeRegion
   }
 }
 
 export const findMachineType = name => {
   return _.find({ name }, machineTypes) || { name, cpu: '?', memory: '?', price: NaN, preemptiblePrice: NaN }
 }
 
-export const getValidGpuTypes = (numCpus, mem) => {
-  const validGpuTypes = _.filter(({ maxNumCpus, maxMem }) => numCpus <= maxNumCpus && mem <= maxMem, gpuTypes)
+export const getValidGpusForZone = zone => {
```

Comment:
Should this be
```suggestion
export const getValidGpuTypesForZone = zone => {
```
?

Similarly for `validGpusForZone` below.

Reply:
Yes, good catch. Done.